imports/api/accounts/methods.js

```diff
@@ -121,6 +94,67 @@ export const remove = new ValidatedMethod({
     }
 });
 
+
+
+export const insertAccountOnSignUp = new ValidatedMethod({
+    name: 'insertAccountOnSignUp',
+    mixins : [LoggedInMixin],
+    checkLoggedInError: {
+        error: 'notLogged',
+        message: 'You need to be logged in to update account'
+    },
+    validate: new SimpleSchema({
+        'userId': {
+            type: Object
+        }
+    }).validator(),
+    run({ userId }) {
+        Accounts.insert({owner :userId, name :'Default' , purpose:'Bank Account' , icon:'abc'  });
+    }
+});
+
+
+
+export const insertAccountOnLogin = new ValidatedMethod({
+    name: 'insertAccountOnLogin',
+    mixins : [LoggedInMixin],
+    checkLoggedInError: {
+        error: 'notLogged',
+        message: 'You need to be logged in to update account'
+    },
+    validate: new SimpleSchema({
+        'userId': {
+            type: Object
+        }
```

Comment:
Also create validation for `id` which is coming in `userId` object. Check this for each method.